tests/unit/models/cart-model-test.js

```diff
@@ -96,6 +96,29 @@ test('it creates a line item when you add a variant', function (assert) {
   });
 });
 
+test('it creates a line item when you add a variant', function (assert) {
+  assert.expect(3);
+
+  const done = assert.async();
+
+  const quantity = 2;
+
+  const variant = singleProductFixture.product_listing.variants[1];
+
```

Comment:
So is it good now? If we take the initial quantity and ensure it has increased it should be good.